department-of-veterans-affairs/caseflow
# VLJ # in spreadsheet instead of css_id

This PR updates the spreadsheet so accept VLJ ids instead of css ids.

To test:
- Import spreadsheet with VLJ ids
- Confirm there are no errors
- Confirm the database still stores CSS ids
